index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Listing</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://xpressui1.netlify.app/xPressUi.css">
```

Comment:
You can import the component library link directly in the main CSS.
By doing this you will not have to give links in all HTML files.